MSAL/test/unit/native_auth/mock/MSALNativeAuthFactoriesMocks.swift

```diff
@@ -43,16 +43,12 @@ class MSALNativeAuthResultFactoryMock: MSALNativeAuthResultBuildable {
         return makeAccount ?? MSALAccount.init(msidAccount: tokenResult.account, createTenantProfile: false)
     }
 
-    func mockMakeNativeAuthTokens(_ authTokens: MSALNativeAuthTokens) {
-        self.makeNativeAuthTokens = authTokens
+    func mockMakeNativeAuthRawIdToken(_ rawIdToken: String) {
```

Comment:
Both functions `mockMakeNativeAuthRawIdToken()` and `makeRawIdToken()` are not used anywhere, can we remove them (along with the variable `makeNativeRawIdToken`)?

Reply:
Done.